src/Microsoft.AspNet.Server.Kestrel/Http/Frame.cs

```diff
@@ -248,8 +245,7 @@ public void Abort()
                         ResponseBody = _responseBody;
                         DuplexStream = new FrameDuplexStream(RequestBody, ResponseBody);
 
-                        _requestAbortCts = CancellationTokenSource.CreateLinkedTokenSource(_disconnectCts.Token);
-                        RequestAborted = _requestAbortCts.Token;
+                        RequestAborted = _disconnectOrAbortedCts.Token;
```

Comment:
K, might be looking at this wrong. Does the `_disconnectCts` do anything particularly beyond the `_requestAbortCts`?